meadServer/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
`selected: ''`

Kind of weird to test for an empty string everywhere instead of just setting the base case to undefined